website/docs/cloud-docs/workspaces/variables/index.mdx

```diff
@@ -152,3 +152,25 @@ When you trigger a run through the command line, Terraform Cloud applies the fol
 - **C_Variable_Set:** `KEY2`. For this run, `KEY2` equals `c`. This is a global variable set with no overwritten values.
 
 ![An example scenario demonstrating variable precedence in Terraform Cloud](/img/docs/variable-precedence-example.png)
+
+
+## Precedence with Priority Variable Sets
+
+You can select to prioritize the values of variables in a variable set.
+When a variable set is prioritized the values take precedence over any variables with the same key set at a more specific scope.
+
+* A project scope is more specific than a global scope.
+* A workspace variable set scope is more specific than a project scope.
+* A workspace variable scope is more specific than a workspace variable set scope.
+* A variable set on the command line a more specific scope than a workspace variable.
```

Comment:
```suggestion
The values in a priority variable set overwrite any other variables with the same key. 
```

it seems like there are no exceptions to this, so i wonder if we can just say "they overwrite everything"? 

if we update the precedence list above, i think we can skip talking about scope specificity here to avoid redundancy

Reply:
A user could in theory mark multiple variable sets as priority and the precedence that would those varsets would follow is the above.

Perhaps a compromise is "The values in a priority variable set overwrite any other variables with the same key set with a more specific scope". ?

Reply:
I think the only thing to note here is that a priority variable set configured to apply to only some projects would be overwritten by a priority variable set configured globally, so maybe we still. need to indicate that there is still some form of precedence